source/Camera/Advanced/CaptureHDRPrintNormals/CaptureHDRPrintNormals.cpp

```diff
@@ -0,0 +1,71 @@
+/*
+Capture Zivid point clouds, compute normals and print a subset.
+
+For scenes with high dynamic range we combine multiple acquisitions to get an HDR point cloud.
+*/
+
+#include <Zivid/Zivid.h>
+
+#include <iostream>
+
+namespace
+{
+    void printNormals(int radius,
+                      const Zivid::Array2D<Zivid::NormalXYZ> &normals,
+                      const int num_of_rows,
+                      const int num_of_cols)
+    {
+        const auto lineSep = std::string(50, '-');
```

Comment:
Not entirely satisfied with the name of the variable, but I know it is standard and known (in python at least). I would spell it out completely.